docs/getting-started/install.rst

```diff
@@ -68,42 +68,42 @@ using `pip`_:
 
 .. code-block:: bash
 
-   $ python -m pip install gammapy
+   python -m pip install gammapy
 
 This will install Gammapy with the required dependencies only.
     
 To install Gammapy with all optional dependencies, you can specify:
 
 .. code-block:: bash
 
-   $ python -m pip install gammapy[all]
+   python -m pip install gammapy[all]
 
 
 To update an existing installation use:
 
 .. code-block:: bash
 
-   $ python -m pip install gammapy --upgrade
+   python -m pip install gammapy --upgrade
 
 To install a specific version of Gammapy use:
 
 .. code-block:: bash
 
-    $ python -m pip install gammapy==1.0
+    python -m pip install gammapy==1.0
 
 To install the current Gammapy **development** version with `pip`_ use:
 
 .. code-block:: bash
 
-   $ python -m pip install git+https://github.com/gammapy/gammapy.git#egg=gammapy
+   python -m pip install git+https://github.com/gammapy/gammapy.git#egg=gammapy
 
 If you want to study or edit the code locally, use the following:
 
 .. code-block:: bash
 
-   $ git clone https://github.com/gammapy/gammapy.git
-   $ cd gammapy
-   $ python -m pip install .
+   git clone https://github.com/gammapy/gammapy.git
```

Comment:
Any particular reason why you removed the $ signs? Especially for blocks that contain multiple commands they are more clear I think